packages/firestore/src/local/simple_db.ts

```diff
@@ -326,10 +326,11 @@ export class SimpleDb {
             reject(
               new FirestoreError(
                 Code.FAILED_PRECONDITION,
-                'Unable to open an IndexedDb connection; this may be due to running in a ' +
-                  'Firefox private browsing session, in which IndexedDb is not supported ' +
-                  '(https://goo.gle/3ymSNyf): ' +
-                  error
+                'Unable to open an IndexedDb connection. This could be due to running in a ' +
```

Comment:
The official spelling seems to be IndexedDB.

Reply:
Done.